catalog/ui/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js

```diff
@@ -80,6 +81,10 @@ require([
         }
 
         var response = getErrorResponse(event, jqxhr, settings, throwError);
+        if (properties.disableUnknownErrorBox && response.message.substring(0,7) === "Unknown") {
```

Comment:
I am torn... don't love the text matching, but I am guessing you did not want put the check up on the default for getErrorResponse then have a magic "null" or some other value to know to return.  I can live with how it is.

Reply:
I think using the more explicit `Unknown Error` string would be better here. I can imagine other types of unknowns possibly being used down the line like unknown source or something that we don't want to catch in this conditional

Reply:
changed to check for `Unknown Error`